components/movementsensor/gpsrtkpmtk/gpsrtkpmtk_test.go

```diff
@@ -0,0 +1,189 @@
+package gpsrtkpmtk
+
+import (
+	"context"
+	"math"
+	"testing"
+
+	"github.com/edaniels/golog"
+	geo "github.com/kellydunn/golang-geo"
+	"go.viam.com/test"
+	"go.viam.com/utils"
+
+	"go.viam.com/rdk/components/movementsensor/fake"
+	rtk "go.viam.com/rdk/components/movementsensor/rtkutils"
+)
+
+var (
+	alt   = 50.5
+	speed = 5.4
+)
+
+const (
+	testRoverName   = "testRover"
+	testStationName = "testStation"
+	testBoardName   = "board1"
+	testBusName     = "bus1"
+	testi2cAddr     = 44
+)
+
+func TestValidateRTK(t *testing.T) {
+	path := "path"
+	fakecfg := &Config{
+		Board:                testBoardName,
+		I2CBus:               testBusName,
+		I2CAddr:              testi2cAddr,
+		I2CBaudRate:          4400,
```

Comment:
I would change to a valid baud rate just so theres no confusion, also might be worth adding in the check for a valid baud rate in the validate using the utils helper function